docs/getting-started-guides/docker.md

```diff
@@ -55,6 +55,25 @@ Here's a diagram of what the final result will look like:
 ### Prerequisites
 
 1. You need to have docker installed on one machine.
+2. Your kernel should support memory and swap accounting. Ensure that the
```

Comment:
This is related to issue #9837 . I guess the root-cause is in docker, but the problem is that docker works fine even without this config, but kubernets starts failing to launch the master container.